resources/js/Components/ClubPage/ReportButton.tsx

```diff
@@ -0,0 +1,13 @@
+import ReportSVG from "@/Components/ClubPage/SVG/ReportSVG";
+
+export default function ReportButton() {
+    return (
+        <a
```

Comment:
Alle button hebben dezelfde code buiten
- Label
- url

Kun je zoiets maken

```js

export function GenericButton(label: string, url: string) {
    return <a href={url} className="..."> {label} </a>
}

export function LeaveReviewButton() {
    return GenericButton("LEAVE AN REVIEW", "/review")
}

...

```

Reply:
is dat niet de onclick die ervoor zorgt dat de button iets andres doet

Reply:
Wat bedoel je?

Reply:
Ah, dan hoeft er geen href te zijn. En zou je een button moeten gebruiken. 

```
<GenericButton label={...} onClick={} />
```